components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java

```
@@ -384,6 +388,10 @@ private void printUsage() {
       "      -no-sequential: do not assume that planes are written in sequential order",
       "               -swap: override the default input dimension order; argument is f.i. XYCTZ",
       "               -fill: byte value to use for undefined pixels (0-255)",
+      "      -precompressed: transfer compressed bytes from input dataset directly to output.",
+      "                      Most input and output formats do not support this option.",
+      "                      Do not use -crop, -fill, or -autoscale, or pyramid generation options",
```

Is `or -autoscale` an oversight as the current converter deals correctly with `-noflat -precompressed -autoscale` ?

---

Good catch. With 0049264045, using `-autoscale` with `-precompressed` should throw an exception before conversion starts. `-autoscale` is meant to change pixel values (and potentially pixel type), so is incompatible with using precompressed tiles directly from the input file.